Detectors/FIT/FV0/simulation/include/FV0Simulation/DigitizationParameters.h

```diff
@@ -0,0 +1,37 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef ALICEO2_FV0_DIGITIZATION_PARAMETERS
+#define ALICEO2_FV0_DIGITIZATION_PARAMETERS
+
+namespace o2::fv0
+{
+struct DigitizationParameters {
+  const Int_t NCellsA = 40;
+  const Int_t mPmts = 48;                       // number of PMTs
+  const Float_t mIntrinsicTimeRes = 0.91;       // FDD 0.44
+  const Float_t mPhotoCathodeEfficiency = 0.23; // quantum efficiency = nOfPhotoE_emitted_by_photocathode / nIncidentPhotons
+  const Float_t mLightYield = 0.1;              // TODO: light collection efficiency to be tuned using collision data
+  const Float_t mPmtGain = 4.21e6;              // for PMT R5924-70 at 0.5T
```

Comment:
... but are these parameters really constants? I suspect at least some of them should be settable? You can use the [ConfigurableParam, e.g.](https://github.com/AliceO2Group/AliceO2/blob/dev/Detectors/ZDC/simulation/include/ZDCSimulation/ZDCSimParam.h) to have them settable from the command line or ini file (eventually from the ccdb).

Reply:
I have changed all those that are true constants to ahere to coding conventions. The rest that may eventually be taken from CCDB are still in camel case. Again I'd prefer to implement the ConfigurableParam as a next step.